src/java/org/apache/cassandra/hadoop/ConfigHelper.java

```diff
@@ -233,9 +233,9 @@ private static String thriftToString(TBase object)
     {
         assert object != null;
         // this is so awful it's kind of cool!
-        TSerializer serializer = new TSerializer(new TBinaryProtocol.Factory());
```

Comment:
Nit: we probably wanna move this useful comment inside the try statement as well